lib/rules/space-after-keywords.js

```diff
@@ -23,6 +23,10 @@ module.exports = function(context) {
      * @returns {void}
      */
     function checkTokens(node, left, right) {
+        if (right.type !== "Punctuator") {
```

Comment:
This could cause duplicate warnings with brace-style. What happens on line 38 if we add a check whether the right token is an opening paren?